admin/release.py

```diff
@@ -263,8 +265,9 @@ class PublishDocsOptions(Options):
          "The version of flocker from which the documentation was built."],
         ["doc-version", None, None,
          "The version to publish the documentation as.\n"
-         "This will differ from \"flocker-version\" for staging uploads and "
-         "documentation releases."],
+         "This will differ from \"flocker-version\" for staging uploads\n"
+         "Attempting to publish documentation as a documentation version "
+         "publishes it as the version being updated.\n"],
     ]
```

Comment:
Again, the proliferation of the words "documentation" and "version" makes this hard to understand.